definition_schema.json

```diff
@@ -48,30 +48,104 @@
     "_abbrev": {
       "type": "string"
     },
-    "_examples": {
-      "type": "array"
-    },
-    "_history": {
-      "type": "array"
-    },
-    "_links": {
-      "type": "object"
-    },
     "_name": {
       "type": "string"
     },
     "_version": {
       "type": "string"
     },
-    "properties": {
-      "type": "object"
-    },
     "additionalProperties": {
       "type": "boolean"
     },
     "_format": {
       "type": "string"
     },
+    "_examples": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "title": {
+            "type": "string"
+          },
+          "url": {
+            "type": "string",
+            "format": "uri-reference"
+          }
+        },
+        "required": [
+          "title",
+          "url"
+        ],
+        "additionalProperties": false
+      }
+    },
+    "_history": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "version": {
+            "type": "string"
+          },
+          "introduced_in": {
+            "type": "string"
+          },
+          "changes": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "version"
+        ]
+      }
+    },
+    "_links": {
+      "type": "object",
+      "patternProperties": {
+        "^[A-Z_]+$": {
+          "type": "object",
+          "properties": {
+            "description": {
+              "type": "string"
+            },
+            "required": {
+              "type": "boolean"
+            },
+            "multiple": {
+              "type": "boolean"
+            },
+            "targets": {
+              "type": "object",
+              "properties": {
+                "any_type": {
+                  "type": "boolean"
+                },
+                "types": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  }
+                }
+              },
+              "required": [
+                "any_type",
+                "types"
+              ]
+            }
+          },
+          "required": [
+            "description",
+            "required",
+            "multiple",
```

Comment:
According to https://github.com/eiffel-community/eiffel/blob/master/eiffel-syntax-and-usage/event-schemas.md, `required` and `multiple` are optional.

Reply:
Thanks for spotting the difference. I couldn't think of any reason for not making them mandatory so I updated event-schemas.md instead in [15ac806](https://github.com/eiffel-community/eiffel/pull/402/commits/15ac806fc609fe728f0291087b2a13e9c17d3602).  Please disagree with me here if you think it is wrong.